dts/bindings/led/issi,is31fl3216a.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2023 Endor AG
+# SPDX-License-Identifier: Apache-2.0
+
+description: Lumissil Microsystems IS31FL3216A
```

Comment:
Please let appear the `ISSI` string as well.

Reply:
Alright, will do!